lightning.js

```diff
@@ -1,8 +1,11 @@
 // setup lnd rpc
 const config = require('./config');
 var fs = require('fs');
-var grpc = require('grpc');
-var lnrpc = grpc.load('rpc.proto').lnrpc;
+var grpc = require('@grpc/grpc-js');
+const protoLoader = require('@grpc/proto-loader');
+const packageDefinition = protoLoader.loadSync('rpc.proto', {});
+var lnrpc = grpc.loadPackageDefinition(packageDefinition).lnrpc;
```

Comment:
gj

Reply:
Sorry for the dumb question, what does "gj" mean?

Reply:
good job